run_service_ssh.ps1

```diff
@@ -21,6 +21,8 @@ $ggp_path = "$($ggp_sdk_path)dev\bin\ggp.exe"
 IF($deploy) {
     $ggp_put_command = "`"$($ggp_path)`" ssh put `"$($deploy)`""
     Invoke-Expression "& $ggp_put_command"
+	$ggp_chmod_command = "`"$($ggp_path)`" ssh shell -- chmod u+x /mnt/developer/OrbitService"
```

Comment:
Can you replace the tab by spaces to make it aligned?

Reply:
thanks, done